### DISCUSSION TITLE
Adding a code to disarm

### DISCUSSION THREAD

**Discussion by designasaw** [fuatakgun/eufy_security#619] [General]
Is it possible to set up a code to disarm the homebase? Done some googling which seems to show the standard HA alarm panel with a code option but mine doesn't and i'm unsure why. Looking at the documentation for the add on i've not seen anything about adding a disarm code. 

Can someone point me in the right direction? 

Adding a wall mount tablet near the front door so would like a bit of "security" to disarm the cameras.

**Comment by fuatakgun** [comment]
No, not possible, you have to define your own template alarm panel in home assistant on top of eufy security based one.

**Reply by KillaDreams** [reply]
Put a screen lock on the tablet... then everything is secure and nobody can change anything
